uis/src/com/biglybt/ui/swt/mainwindow/MainStatusBar.java

```diff
@@ -613,7 +613,7 @@ public void handleEvent(Core core, Event e) {
 						ConfigSection.SECTION_CONNECTION);
 
 				if (PluginInitializer.getDefaultInterface().getConnectionManager().getNATStatus() != ConnectionManager.NAT_OK) {
-					Utils.launch(Constants.AZUREUS_WIKI + "NAT_problem");
```

Comment:
There are many wiki pages that exists on the official vuze wiki, but doesn't exits on the new BiglyBT wiki.

What is `Constants.AZUREUS_WIKI` ? The current implementation references to https://wiki.biglybt.com/w/NAT_problem
which doesn't exists. Is that constant deprecated or is this instance supposed to link to https://wiki.vuze.com/w/NAT_problem ?

Reply:
Due to legal risks that we aren't willing to take, we try not to link to any vuze.com URLs.  The fact that the Google results brings up vuze.com URLs is completely out of our control :)

We hope that over time, people would write new wiki pages specifically for BiglyBT.

Reply:
This field should be `@Depricated` then and possibly removed. 
I'm not going to suggest removing it since it's part of the public api and i have no idea about it's usage. 
- Is there any repository or project that has all the different plugins linked in as source code?

Reply:
Deprecated definitely makes sense.  I'm just guessing, but the reason it's probably still around is it was just easier to change the constant value than to replace the constant name.  That, and I think the "w/" is what makes our Google Search redirect work.

Reply:
If someone could guarantee it's safe to delete it I could. That would be cleaner....

Reply:
It should definitely not be deleted - when a page doesn't exist (yet) on the BiglyBT wiki we show a general google search result for the terms involved. This quite often leads to the top hit being quite relevant - open https://wiki.biglybt.com/w/NAT_problem in google for example.

Reply:
Sorry, I realize I wasn't spesific enough. My bad. 

What I believe is the `Constants.AZUREUS_WIKI` to be a duplicate of `Constants.URL_WIKI`. The only difference between those two is the first has the "w/" path built in. @parg , is there any reason to hold on to the `AZUREUS_WIKI` constant for plugin compatibility?

Reply:
Ah, my misunderstanding! 

I have a large number of the 'real' plugins loaded in Eclipse and none of them refer to AZUREUS_WIKI - so go for it and remove it, I'll take the flak if it breaks anything!

Reply:
I added a commit removing the AZUREUS_WIKI constant.